grouper/fe/templates/group-requests.html

```diff
@@ -48,7 +48,12 @@
                         <td class="request-requester">{{ request.requester }}</td>
                         <td class="request-status">{{ request.status }}</td>
                         <td class="request-requested-at">{{ request.requested_at|print_date }}</td>
-                        <td class="request-expiration">{{ request.changes['expiration'] }}</td>
+                        {% if request.changes.expiration == '' or request.changes.expiration is none or request.changes.expiration is not defined%}
```

Comment:
Can we simplify this, maybe to something like `request.changes.get('expiration')` and let `print_date` handle the `None`?